common/lib/xmodule/xmodule/x_module.py

```diff
@@ -39,6 +39,7 @@
 
 XMODULE_METRIC_NAME = 'edxapp.xmodule'
 XMODULE_DURATION_METRIC_NAME = XMODULE_METRIC_NAME + '.duration'
+XMODULE_METRIC_SAMPLE_RATE = 0.1
```

Comment:
I don't think it's likely to be an operational issue, so we probably don't need to change it terribly rapidly.

@feanil: what say you? Should I pull this out of configuration (either file-backed, or database-backed)?